docs/docs/connecting-to-eventstore/connecting-to-a-cluster.md

```diff
@@ -0,0 +1,64 @@
+# Connecting to a cluster
+
+::: tip
+Packages required
+- EventStore.Client.Grpc.Streams
+:::
+
+TODO: Add link to how to set up cluster
+
+## Simple connection
+
+For redundancy, you can run EventStoreDB in a cluster. In this scenario you should specify all the nodes in your cluster when connecting. This allows your client to discover which node it should be speaking to at any given time.
+
+::: warning
+Important: You should never hide your cluster behind a load balancer as this will stop your clients from connecting to the correct nodes.
+:::
+ 
+<<< @/samples/connecting-to-a-cluster/Program.cs#connecting-to-a-cluster
+ 
+## Fine-tuning cluster connection
+
+There are a number of additional settings that can be altered when connecting to a cluster.
+
+<<< @/samples/connecting-to-a-cluster/Program.cs#connecting-to-a-cluster-complex
+
+### DiscoveryInterval (Default: 100ms)
+
+The interval between node discovery attempts. If the client attempts to discover a node and it is unresponsive this is the length of time the client will wait before trying to discover it again.
+
+### GossipTimeout (Default: 10sec)
+
+The length of time the client will wait to get a gossip request from a node. The gossip requests lets the client know the current state of any given node. To find out more see 
```

Comment:
```suggestion
The length of time the client will wait to get a gossip request from a node. The gossip requests let the client know the current state of any given node. To find out more see 
```

Reply:
Thanks for the feedback @bartelink much appreciated!